$OpenBSD: patch-include_xcb_compat_h,v 1.2 2012/01/20 20:16:32 dcoppa Exp $

Fix xcb_compat.h to make i3 compile with older xcb versions
(upstream git commit 07e495d2d0a2e6d7cf18f96d5584abd8b1e5dd56)

--- include/xcb_compat.h.orig	Fri Jan 20 15:10:01 2012
+++ include/xcb_compat.h	Fri Jan 20 15:10:52 2012
@@ -27,6 +27,7 @@
 #define XCB_ICCCM_SIZE_HINT_BASE_SIZE XCB_SIZE_HINT_BASE_SIZE
 #define XCB_ICCCM_SIZE_HINT_P_ASPECT XCB_SIZE_HINT_P_ASPECT
 #define xcb_icccm_wm_hints_t xcb_wm_hints_t
+#define xcb_icccm_get_wm_hints xcb_get_wm_hints
 #define xcb_icccm_get_wm_hints_from_reply xcb_get_wm_hints_from_reply
 #define xcb_icccm_get_wm_hints_reply xcb_get_wm_hints_reply
 #define xcb_icccm_get_wm_hints_unchecked xcb_get_wm_hints_unchecked
